src/Synth/LFO.cpp

```diff
@@ -154,6 +159,22 @@ float LFO::biquad(float input)
     return (cutoff==127) ? input : output; // at cutoff 127 bypass filter
 }
 
+void LFO::releasekey()
+{
+    if (lfopars_.fadeout==10.0f) // deactivated
+        return;
+    // store current ramp value in case of release while fading in
+    rampConst = ramp; 
+    outConst *= ((1.0f/ramp)-1.0f);
```

Comment:
I don't understand this statement (maybe comment it?). Also, it might end up with division by zero?

Reply:
> Btw, we have a segmentation fault in the watch test (with guitar-adnote.xmz).

i don't know how to debug them

Reply:
> I don't understand this statement (maybe comment it?). Also, it might end up with division by zero?

i removed the division and commented it.

Reply:
segfaults fixed